src/components/pages/MyCases/MyCases.js

```diff
@@ -216,16 +201,16 @@ export default function MyCases(props) {
               />
             </svg>
           }
+          onClick={e => {
+            e.preventDefault();
+          }}
         ></Button>
       ),
     },
     {
-      field: 'status',
-      renderHeader: params => <strong>{'Status'}</strong>,
-      headerName: 'Status',
-      headerAlign: 'center',
-      flex: 1,
-      renderCell: params => <span>Approved</span>,
+      key: 'status',
+      title: row => <strong>{'Status'}</strong>,
+      render: row => <span>Approved</span>,
```

Comment:
Just a heads up in case of merge conflict, I added the progress bar component to show a completed status!